src/cljs/ataru/hakija/application_handlers.cljs

```diff
@@ -79,13 +79,7 @@
 (reg-event-fx
   :application/submit
   (fn [{:keys [db]} _]
-    (send-application db :submit)
```

Comment:
So all of this was removed, does it still work? Did we only remove the submitting-status?